test_programs/compile_success_empty/reexports/src/main.nr

```diff
@@ -1,4 +1,4 @@
-use reexporting_lib::{FooStruct, MyStruct, lib};
+use dep::reexporting_lib::{FooStruct, MyStruct, lib};
```

Comment:
Is this a regression?

Reply:
I don't have strong feelings on this now tbh. `dep::` is more explicit so 🤷

Reply:
I won't block for this, but just wanted to note. Are we not looking to eventually error on `dep::`?